CHANGELOG.md

```diff
@@ -2,10 +2,42 @@
 
 ## Unreleased
 
-- [Pull request #601: Add input classes to implement a part of #428](https://github.com/alphagov/accessible-autocomplete/pull/601)
-- [Pull request #621: Handle query change when the text length does not change](https://github.com/alphagov/accessible-autocomplete/pull/621)
-- [Pull request #620: Add `className` attribute to status component](https://github.com/alphagov/accessible-autocomplete/pull/620)
-- [Pull request #591: Add menuAttributes to fix #361](https://github.com/alphagov/accessible-autocomplete/pull/591)
+### Breaking changes
+
+#### Verify your code does not rely on polyfills we have now removed
+
+The code of the accessible autocomplete is no longer transpilled and polyfilled to support IE8-10, matching [our previous announcement](https://github.com/alphagov/design-system-team-internal/issues/561).
```

Comment:
```suggestion
Following on from [our previous announcement](https://github.com/alphagov/design-system-team-internal/issues/561), the accessible autocomplete code is no longer transpilled and polyfilled to support IE8-10.
```

Reply:
Think there's one too many l's in transpiled?

Reply:
'tis a better pil than a bitter pill